opba-embedded-starter/src/main/resources/logback.xml

```diff
@@ -4,11 +4,10 @@
             <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{50} - %msg%n</pattern>
         </encoder>
     </appender>
-    <!-- please do not remove but replace log level, thx peter -->
-    <logger name="de.adorsys.opba.tppbankingapi.mapper.generated.AccountListFacadeToRestMapper" level="WARN"/>
-    <logger name="de.adorsys.opba.tppbankingapi.mapper.generated.TransactionsFacadeToRestMapperImpl" level="WARN"/>
-    <logger name="de.adorsys.opba.protocol.bpmnshared.service.context.ContextUtil" level="WARN"/>
-    <logger name="de.adorsys.opba.protocol.xs2a.service.mappers.generated.Xs2aResultBodyExtractor$Xs2aToFacadeMapperImpl" level="WARN"/>
+
+    <logger name="de.adorsys.opba.tppauthapi.controller.PsuAuthController" level="debug"/>
```

Comment:
Do not commit such log levels, please revert back to WARN

Reply:
ok